narwhals/expression.py

```diff
@@ -410,6 +410,56 @@ def filter(self, other: Any) -> Expr:
         )
 
     def is_null(self) -> Expr:
+        """
+        Returns a boolean Series indicating which values are null.
+
+        Examples:
+            >>> import pandas as pd
+            >>> import polars as pl
+            >>> import narwhals as nw
+            >>> df_pd = pd.DataFrame(
+            ...         {
+            ...             'a': [2, 4, None, 3, 5],
+            ...             'b': [2.0, 4.0, float("nan"), 3.0, 5.0]
+            ...         }
+            ... )
+            >>> df_pl = pl.DataFrame(
+            ...         {
+            ...             'a': [2, 4, None, 3, 5],
+            ...             'b': [2.0, 4.0, float("nan"), 3.0, 5.0]
+            ...         }
+            ... )
+
+            Let's define a dataframe-agnostic function:
+
+            >>> def func(df_any):
+            ...     df = nw.from_native(df_any)
+            ...     df = df.with_columns(nw.all().is_null()) # nan != null for polars
```

Comment:
how about showing:
```python
...     df = df.with_columns(
...         a_is_null = nw.col('a').is_null(),
...         b_is_null = nw.col('b').is_null(),
...     )
```
, so that the output is easier to reason about?

Reply:
Yeah, that makes more sense. I'll add it in the next pull request

Reply:
thanks! can you add it to this one please? if you make your change, add and commit them, and push them to this branch, then this pull request will be automatically updated

Reply:
Okay, I've updated the branch now